extensions_admin/pulp_docker/extensions/admin/upload.py

```diff
@@ -30,12 +30,16 @@ def __init__(self, context):
 
     def determine_type_id(self, filename, **kwargs):
         """
-        We only support one content type, so this always returns that.
+        Determine whether it is a V1 or V2 image
 
         :return: ID of the type of file being uploaded
         :rtype:  str
         """
-        return constants.IMAGE_TYPE_ID
+        image_manifest = tarutils.get_image_manifest(filename)
+        if isinstance(image_manifest, list):
+            return constants.IMAGE_TYPE_ID
+        elif image_manifest['schemaVersion'] == constants.S2_VERSION:
```

Comment:
and the same highlight as mentioned above, no need to check for schema version since skopeo supports just schema2 for now, and even if they would add support for schema1, there would be enough to check just for presence of this key, because in v1 image. there is not such concept.